nativelib/src/main/scala/java/lang/Class.scala

```diff
@@ -144,8 +149,32 @@ final class _Class[A] {
   ): java.lang.reflect.Method = ???
   @stub
   def getMethods(): Array[Method] = ???
-  @stub
-  def getResourceAsStream(name: java.lang.String): java.io.InputStream = ???
+
+  def getResourceAsStream(name: java.lang.String): java.io.InputStream = {
+    val absoluteName =
+      if (name(0) == '/') {
```

Comment:
I tried to see what happens on JVM and it turns out that something interesting does. All file names from the specified directory in a jar (so here relative ".") are put to an InputStream and returned. I'll try to implement that for the embedded files. For our use case I fear that there wouldn't be much use for encoding source paths like this, since we cannot access them either way

Reply:
Turns out this doesn't happen on JVM when compiled into a JAR, so I ended up not adding that feature